src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Listener.cs

```diff
@@ -77,7 +127,22 @@ private static void ConnectionCallback(UvStreamHandle stream, int status, Except
         /// </summary>
         /// <param name="listenSocket">Socket being used to listen on</param>
         /// <param name="status">Connection status</param>
-        protected abstract void OnConnection(UvStreamHandle listenSocket, int status);
+        private void OnConnection(UvStreamHandle listenSocket, int status)
+        {
+            UvStreamHandle acceptSocket = null;
+
+            try
+            {
+                acceptSocket = CreateAcceptSocket();
+                listenSocket.Accept(acceptSocket);
+                DispatchConnection(acceptSocket);
+            }
+            catch (UvException ex)
+            {
+                Log.LogError(0, ex, "Listener.OnConnection");
```

Comment:
I tend to prefer `nameof` for things like this because it's easy to miss that if there's a rename.